### PR TITLE
API-34536 POA v2: Request schema cleanup

### PR DESCRIPTION
**Pull request by dellerbie**
## Summary
Clean up items to ensure alignment with the related forms.

**2122 - Update Request Schema**

- add veteran.serviceNumber
- remove veteran.serviceBranch
- remove signatures object
- consentLimits
  - update enum values to use CAPS_WITH_UNDERSCORE format

**2122a - Update Request Schema**

- remove veteran.vaFileNumber
- veteran.serviceBranch
- update enum values to use CAPS_WITH_UNDERSCORE format
- add veteran.serviceBranchOther
  - string value for writing in if serviceBranch = OTHER
  - max chars 27
  - also needs to be added to the form builder
- representative.type
  - remove 'INDIVIDUAL PROVIDING REPRESENTATION UNDER SECTION 14.630'
  - remove 'SERVICE ORGANIZATION REPRESENTATIVE'
- consentLimits
  - update enum values to use CAPS_WITH_UNDERSCORE format
  - updated enum values will require update to logic for mapping

## Related issue(s)
[API-34536](https://jira.devops.va.gov/browse/API-34536)

## Screenshots
Screenshot 2024-03-13 at 10 07 31 AM

Screenshot 2024-03-13 at 10 08 24 AM

## What areas of the site does it impact?
Lighthouse Claims API v2

## Acceptance criteria
- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature (Documentation updates)

### CODE REVIEW

File: modules/claims_api/config/schemas/v2/2122.json
Comment (jvcAdHoc) [inline modules/claims_api/config/schemas/v2/2122.json:1]: Not sure if Austin is handling this in the signatures tickets, but if not, `sigantures` shouldn't be in the schema

File: modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb [hunk 1]
```diff
@@ -132,13 +126,7 @@ def page1_options(data)
             # Item 15B
             "#{base_form}.Checkbox1[0]": (data.dig('representative', 'type') == 'ATTORNEY' ? 1 : 0),
             "#{base_form}.Checkbox2[0]": (data.dig('representative', 'type') == 'AGENT' ? 1 : 0),
-            "#{base_form}.Checkbox3[0]": (data.dig('representative', 'type') == 'SERVICE ORGANIZATION REPRESENTATIVE' ? 1 : 0),
-            "#{base_form}.Checkbox[0]": (data.dig('representative', 'type') == 'INDIVIDUAL PROVIDING REPRESENTATION UNDER SECTION 14.630' ? 1 : 0),
             "#{base_form}.SpecifyOrganization[0]": data.dig('representative', 'organizationName'),
-            # Item 16B
-            "#{base_form}.Date_Of_Signature[0]": (data.dig('representative', 'type') == 'INDIVIDUAL PROVIDING REPRESENTATION UNDER SECTION 14.630' ? I18n.l(Time.zone.now.to_date, format: :va_form) : nil),
```
Comment (jvcAdHoc) [inline modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb:139]: we probably don't need `organizationName` with this option being removed

File: modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb
Comment (jvcAdHoc) [inline modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb:1]: With changing the consent limits to `_`, they appear on the 2122a as `DRUG_ABUSE`. I'm not sure if that's how we want those on the final pdf.

File: modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb [hunk 2]
```diff
@@ -29,12 +29,7 @@ def page4_template_path
         #
         # @param data [Hash] Hash of data to add to the pdf
         def sign_pdf_text(data)
-          @page1_path = if data.dig('representative',
-                                    'type') == 'INDIVIDUAL PROVIDING REPRESENTATION UNDER SECTION 14.630'
-                          insert_text_signatures(page1_template_path, data['text_signatures']['page1'])
```
Comment (jvcAdHoc) [inline modules/claims_api/lib/claims_api/v2/poa_pdf_constructor/individual.rb:34]: `individual_page1_signatures` in `modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb` should be safe to remove with this change

File: modules/claims_api/spec/requests/v2/veterans/power_of_attorney_org_request_spec.rb [hunk 3]
```diff
@@ -24,17 +24,13 @@
     end
 
     describe 'submit2122' do
-      b64_image = File.read('modules/claims_api/spec/fixtures/signature_b64.txt')
+      File.read('modules/claims_api/spec/fixtures/signature_b64.txt')
```
Comment (mchristiansonVA) [inline modules/claims_api/spec/requests/v2/veterans/power_of_attorney_org_request_spec.rb:27]: Do we still need this line if the variable to which it was assigned is gone?

Comment (rockwellwindsor-va) [reply modules/claims_api/spec/requests/v2/veterans/power_of_attorney_org_request_spec.rb]: no we do not, good catch fixing it now

**Review by mchristiansonVA — APPROVED** [review]
Looks good!

**Pull request merged**